Client/src/Components/TabPanels/Account/PasswordPanel.jsx

```diff
@@ -44,7 +46,7 @@ const PasswordPanel = () => {
 
     const validation = editPasswordValidation.validate(
       { [name]: value },
-      { abortEarly: false, context: { newpassword: localData.newpassword } }
+      { abortEarly: false, context: { newPassword: localData.newPassword } }
```

Comment:
I just realized I totally ignored our current naming convention here 😂 these body params probably should all be camel case instead of all lower case..
ie firstName instead of firstname 🤔thoughts? Worth changing?

Reply:
True, camel case does seem more appropriate. It's up to you, Captain. You say jump, and I'm diving overboard 😄.

Reply:
🏴‍☠️ Off you go then! Let's leave it for now and I'll open an issue and we can do this refactor when we have a chance